src/Weebly/WeeblyClient.php

```diff
@@ -247,7 +247,11 @@ private function makeRequest($url, $parameters=array(), $method='POST')
     {
         $curl_handler = $this->getCurlHandler();
 
-        if ($method !== 'GET'){
+        if ($method === 'GET'){
```

Comment:
PSR-2 formatting standards require a space after the closing parens here:

`if ($method === 'GET') {`